doofinder-for-woocommerce/includes/class-setup-wizard.php

```diff
@@ -1387,7 +1413,7 @@ private function test_api_settings($api_settings)
 				return true;
 			} catch (\DoofinderManagement\ApiException $exception) {
 				$this->log->log('Wizard Step 2: ' . $exception->getMessage());
-				$this->add_wizard_step_error($step, 'api-endpoint-connection-failed', __('Could not connect to the API. API Key or Host is not valid.', 'woocommerce-doofinder'));
+				$this->add_wizard_step_error(2, 'api-endpoint-connection-failed', __('Could not connect to the API. API Key or Host is not valid.', 'woocommerce-doofinder'));

```

Comment:
Why have we changed this step into it's raw value?

Reply:
Thanks for pointing that.
I will pass the step as an argument to the function to use it instead of that fixed value